script/generate_licenses.py

```diff
@@ -153,7 +153,8 @@ def generate_backgrounds_license(preamble, backgrounds):
 
         filename = validated_data_field(background, 'source')
         author_name = validated_data_field(background, 'author')
-        author_link = validated_data_field(background, 'link')
+        # Don't validate link. it can be empty.
```

Comment:
If there are no links, then could you please update this line:
```
        notices += 'File: %s\nAuthor: %s (%s)\nURL: %s\nLicense: %s\n' \
```
so that the brackets are omitted when the link is empty?

Reply:
Ah,, yes it should be `Author: xxx` instead of `Author:XXX ()`

Reply:
Updated.

Reply:
Thanks @simonhong .